### PR TITLE
Update in-article email sign-up forms

### PR DESCRIPTION
**Pull request by gtrufitt**
- Remove NHS in-article sign-up
- Add Film Today sign-up to film section articles
- Add The Fiver sign-up to articles with the Football keyword

image

image

cc/ @crifmulholland

### CODE REVIEW

**Comment by crifmulholland** [comment]
lovely! :+1:

**Comment by NathanielBennett** [comment]
LGTM :+1:

File: static/src/javascripts/projects/common/modules/email/email-article.js [hunk 1]
```diff
@@ -125,19 +135,22 @@ define([
...
+                return canRunHelpers.keywordExists(['Football']) && canRunHelpers.allowedArticleStructure();
+            },
             theGuardianToday: function () {
                 var host = window.location.host,
                     escapedHost = host.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&'), // Escape anything that will mess up the regex
                     urlRegex = new RegExp('^https?:\/\/' + escapedHost + '\/(uk\/|us\/|au\/|international\/)?([a-z-])+$', 'gi'),
                     browser = detect.getUserAgent.browser,
                     version = detect.getUserAgent.version,
-                    pageIsBlacklisted = canRunHelpers.keywordExists(['NHS', 'US elections 2016']);
+                    pageIsBlacklisted = canRunHelpers.keywordExists(['US elections 2016', 'Football']) || config.page.section === 'film';
```
Comment (crifmulholland) [inline static/src/javascripts/projects/common/modules/email/email-article.js]: Football is a section too - not sure it matters though.

**Comment by gtrufitt** [comment]
Cheers!

**Pull request merged**